src/shared/components/Snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,35 @@
+import { Global } from '@emotion/react'
+import React from 'react'
+import Icon from '../Icon'
+import { SnackbarButton, SnackbarParagraph, snackbarTransitions, SnackbarWrapper } from './Snackbar.style'
+
+export type SnackbarProps = {
+  variant?: 'error' | 'success' | 'info'
+  message: string
+  buttonText?: string
+  onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+}
+
+const SnackbarComponent: React.ForwardRefRenderFunction<HTMLDivElement, SnackbarProps> = (
+  { variant = 'info', message, buttonText = 'Ok', onClick },
+  ref
+) => {
+  return (
+    <>
+      <Global styles={snackbarTransitions} />
```

Comment:
Maybe it'd be a better idea to include that as part of the Snackbar context? I'm not sure what are performance implications of mounting/unmounting `Global` continuously.